ci/authn-oidc/policy.yml

```diff
@@ -26,6 +26,8 @@
 
```

Comment:
This file is getting too big. Can we separate this file to `okta-policy.yml`, `keycloak-policy.yml` & `adfs-policy.yml`?
Then we can load the different policies in each section per oidc provider

Reply:
Done :)